lib/notifier.rb

```diff
@@ -0,0 +1,17 @@
+require 'twilio-ruby'
+class Notifier
+  def initialize(config_loader_klass:)
```

Comment:
no reason to use klass pattern here - just pass an instance. Why is Notifier responsible for building new instances?